pep_sphinx_extensions/pep_theme/static/mq.css

```diff
@@ -102,4 +102,55 @@
     h3 {
         page-break-after: avoid;
     }
+    h1 {
+        font-size: 18pt;
+        font-weight: bold;
+        text-align: center;
+    }
+    h2, details > summary {
```

Comment:
Maybe it would make sense to add a class to the details element used for the toc and only address that here?